packages/core/src/DirectoryLoader.ts

```diff
@@ -99,10 +141,11 @@ export abstract class DirectoryLoader {
 			}
 		} else {
 			// Short renaming to avoid type issues
-			const tmpNode = tempNode;
-			nodeVersion = Array.isArray(tmpNode.description.version)
-				? tmpNode.description.version.slice(-1)[0]
-				: tmpNode.description.version;
+
+			this.injectCustomApiCallOption(tempNode.description);
```

Comment:
Agree with removing the injection from inside the loop so that not all node versions get the option, but if we want to inject only to the latest version, then the injection should still be inside the first branch and it's now missing it seems? 

As in, this second branch will only be reached if the node is not versioned, so I expect no versioned node is getting the custom option injected right now. But let me know if I'm misunderstanding.

Reply:
You're right of course, I misunderstood how this method works. I've added`injectCustomApiCallOption` call also for `currentVersionNode`. Which now works as expected:
![Uploading CleanShot 2023-02-02 at 10.51.17@2x.png…
<img width="1342" alt="CleanShot 2023-02-02 at 10 51 36@2x" src="https://user-images.githubusercontent.com/12657221/216292032-ffb93312-ec54-494f-b3d7-c202112b6fd2.png">
]()

Thanks for catching this!